src/cascadia/TerminalControl/TermControlAutomationPeer.h

```diff
@@ -33,6 +33,9 @@ Author(s):
 
 namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 {
+    typedef unsigned long long IdType;
+    constexpr IdType InvalidId = 0;

```

Comment:
Wait is this still needed? If so, should this file be including `IUiaTraceable.hpp` instead?